extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -283,8 +283,13 @@ class AmpAccordion extends AMP.BaseElement {
     // overidden.
     const target = dev().assertElement(event.target);
     const header = dev().assertElement(event.currentTarget);
-    const anchor = closest(target, e => e.tagName == 'A', header);
-    if (anchor === null) {
+    const anchorOrTapTarget = closest(target, e => {
+      return (e.tagName == 'A')
+        || (e.hasAttribute('on')
```

Comment:
nit: Please add paranthesis around the `on` attribute related conditions?

Reply:
Is that too verbose? Or should I just remove the parentheses around `(e.tagName == 'A')` instead? These extra parentheses have no programmatic purpose.

Reply:
Nah you're right let it be. I just find statements like A || if B and B meets some condition to be hard to read. 

But that's on me.